assets/src/stories-editor/helpers/index.js

```diff
@@ -684,9 +684,9 @@ export const calculateFontSize = ( measurer, expectedHeight, expectedWidth, maxF
  */
 export const getPercentageFromPixels = ( axis, pixelValue ) => {
 	if ( 'x' === axis ) {
-		return Math.round( ( pixelValue / STORY_PAGE_INNER_WIDTH ) * 100 );
+		return Number( ( ( pixelValue / STORY_PAGE_INNER_WIDTH ) * 100 ).toFixed( 2 ) );
```

Comment:
Note that this change will likely cause validation errors for existing blocks (including templates).

Reply:
Yes, it totally does. But I am not sure how big of an impact the rounding has anyway, so maybe I can revert it again.

Reply:
It actually might have a noticeable impact within #2310, will test it out.

Reply:
Yes, it does have a noticeable impact there, will add it within that PR.

Reply:
works for me!